### DISCUSSION TITLE
Pimcore 6.8 plugin not working in Pimcore X

### DISCUSSION THREAD

**Discussion by nirbanc2020** [pimcore/pimcore#9321] [Q&A / Support]
Hi

I am trying to install and enable the plugin(Data Completeness) in pimcore X.But, in admin panel while clicking on bundle page, I see the lock symbol after my plugin and also it not working.While on termal by running command bin/console pimcore:bundle:list I see that plugin is installed as well as enabled. Please help.
![project-presales-local-Pimcore](https://user-images.githubusercontent.com/73533273/120626293-25903d80-c480-11eb-8eba-f94e6955cafc.png)
![Screenshot from 2021-06-03 15-14-10](https://user-images.githubusercontent.com/73533273/120626300-26c16a80-c480-11eb-8fde-c096c28a08a9.png)

**Comment by fashxp** [comment]
Hi,
please define 'not working'. 

The lock symbol in admin appears when it is not possible to deactivate the bundle afaik. 

BR
Christian
